spec/models/form526_submission_spec.rb

```diff
@@ -98,72 +98,87 @@
   end
 
   describe 'scopes' do
-    describe 'pending_backup_submissions' do
-      let!(:new_submission) { create(:form526_submission) }
-      let!(:rejected_backup_submission) do
-        create(:form526_submission, :backup_path, backup_submitted_claim_status: 'rejected')
-      end
-      let!(:accepted_backup_submission) do
-        create(:form526_submission, :backup_path, backup_submitted_claim_status: 'accepted')
-      end
-      let!(:backup_submission_a) { create(:form526_submission, :backup_path) }
-      let!(:backup_submission_b) { create(:form526_submission, :backup_path) }
+    let!(:in_process) { create(:form526_submission) }
+    let!(:expired) { create(:form526_submission, :created_more_than_3_days_ago) }
+    let!(:happy_path_success) { create(:form526_submission, :with_submitted_claim_id) }
+    let!(:pending_backup) { create(:form526_submission, :backup_path) }
+    let!(:accepted_backup) { create(:form526_submission, :backup_path, :backup_accepted) }
+    let!(:rejected_backup) { create(:form526_submission, :backup_path, :backup_rejected) }
+    let!(:remediated) { create(:form526_submission, :remediated) }
+    let!(:remediated_and_expired) { create(:form526_submission, :remediated) }
+    let!(:remediated_and_rejected) { create(:form526_submission, :remediated, :backup_path) }
```

Comment:
Should this include the `backup_rejected` trait?

Reply:
if it's remediated, we don't care about rejection. Remediation is treated as an override, so yes we explicitly want to include things that have been rejected from the backup path

Reply:
sorry misread, will include yes good catch